### ISSUE TITLE
Overfitting 

### ISSUE THREAD

**Issue by andriyrizhiy** [HuiZeng/Grid-Anchor-based-Image-Cropping#13]
I trying to implement your paper in tensorflow and run experiments in less models, but i get overfitting in training set. You encountered this problem in your experiment (I did not find this in the paper)?

**Comment by HuiZeng** [comment]
Can you provide more information about overfitting? You mean the training loss is smaller than the validatation/testing loss?

**Comment by andriyrizhiy** [comment]
yes, my training loss smaller and decrease and val loss big and increase, and ACC4/5 in training increases but val is staying in place

**Comment by HuiZeng** [comment]
Please refer to issue[14].

**Comment by HuiZeng** [comment]
Please refer to issue #14
